### ISSUE TITLE
Publish `litps` to `npm`?

### ISSUE THREAD

**Issue by paluh** [Thimoteus/literate-purescript#2]
Maybe we can distribute compiled `litps` with `npm`. What do you think?

**Comment by paluh** [comment]
P.S.
@Thimoteus It seems that we can rip this piece from `pscid` or `pulp`.

**Comment by Thimoteus** [comment]
Yeah, probably worth looking into, though I've never published a binary on npm before.

**Comment by paluh** [comment]
I've not done this before too. It seems that `literate-purescript` is already taken on npm. Do you think that `litps` is good as a package name?

**Comment by paluh** [comment]
Following `pscid` `package.json` and `.npmignore` I've created dirty experimental publish workflow which works fine:

https://github.com/paluh/literate-purescript/commit/cc8c02d07fa93e245ef835f803a9d2d7ff852500

I'm not sure if this is the most beautiful approach as it adds `index.js` into the repo and doesn't bundle anything but copies the whole `./output` dir into published package.

Do we want to take this easy route?

**Comment by paluh** [comment]
P.S.
So `npm install paluh-litps` then `paluh-litps --help` works fine.

**Comment by Thimoteus** [comment]
As long as it's maintainable, it should be fine :man_shrugging:

**Comment by paluh** [comment]
@Thimoteus 
It seems to be maintainable as `pscid` is long standing project already.

1. Should I provide a PR with this solution? 

2. Would you be so kind and publish a package yourself? This process requires a registration and npm authentication but it gives you a full control over package release, publishing and unpublishing.

**Comment by paluh** [comment]
@Thimoteus I don't want to bother you but maybe you can find a moment and consider answering above two questions ;-)

**Comment by Thimoteus** [comment]
Hey, I don't think I'll have time to look into this this week, but I've added a reminder for next weekend, so I'll let you know then. For now, you can make a PR and I'll take a look over the weekend.
